mleap-runtime/src/main/scala/ml/combust/mleap/bundle/ops/feature/HashingTermFrequencyOp.scala

```diff
@@ -20,14 +20,17 @@ class HashingTermFrequencyOp extends MleapOp[HashingTermFrequency, HashingTermFr
     override def store(model: Model, obj: HashingTermFrequencyModel)
                       (implicit context: BundleContext[MleapContext]): Model = {
       model.withValue("num_features", Value.long(obj.numFeatures))
-        .withValue("binary", Value.boolean(obj.binary))
+        .withValue("binary", Value.boolean(obj.binary)).
+        withValue("version", Value.int(obj.version))
```

Comment:
do you think it's worth renaming to `hashUnsafeBytesVersion` so that it's not confused with the version of the transformer?